decidim-system/app/forms/decidim/system/update_organization_form.rb

```diff
@@ -4,108 +4,36 @@
 
 module Decidim
   module System
-    # A form object used to update organizations from the system dashboard.
-    #
-    class UpdateOrganizationForm < Form
-      include TranslatableAttributes
-      include JsonbAttributes
+    class UpdateOrganizationForm < BaseOrganizationForm
+      translatable_attribute :name, String
 
-      mimic :organization
+      validate :validate_organization_name_presence
```

Comment:
I think this isn't needed, as we already have it in `BaseOrganizationForm`. Also in `RegisterOrganizationForm` we don't do this, and we should be consistent between these two. 
```suggestion
```

Reply:
Actually we do not have it in `BaseOrganizationForm`. in that class we have a : `validate_organization_uniqueness`

Indeed, we have a `validate_organization_uniqueness` in both `UpdateOrganizationForm`  and `RegisterOrganizationForm`, but the behavior is different. The update version works with the name as a Hash, but the register version works with name as a plain string.

Reply:
I get the different implementations as during Register is a String (as we have the "I don't know how many languages to translate to" problem) and then during Update is a Hash (as we already know the languages).

What I don't buy is in having the method different method names: IMO they should have the same method name always (`:validate_organization_name_presence`) and they should have the `validate` call + the raise in the Base and then the method different method definitions in the classes that inherit from it. 

Or is there a reason that I don't get why we need to have different names for this validation?

Reply:
I guess that you are confusing things. In RegisterOrganizationForm we have a ‘validate :name, presence: true’ that we cannot use in UpdateOrganizationForm. The method you are commenting about validates that name is present within translated strings. 

the other method in discussion is indented to be used to ensure Uniqueness of organization name.

Reply:
> I guess that you are confusing things. 

I think you're right! Giving it a last look it makes perfect sense how it is. I guess I was mixing the naming in my head (`validate_organization_uniqueness` and `validate_organization_name_presence`).